### PR TITLE
Drop templating with php templates

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | #21, #16 
| License       | MIT

### CODE REVIEW

**Comment by kunicmarko20** [comment]
I guess I should mention in upgrading that renderer now accepts only twig?

**Comment by kunicmarko20** [comment]
Should I add a deprecation PR to 1.x?

File: UPGRADE.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # UPGRADE
+
+UPGRADE FROM 1.x to 2.0
+
+symfony/templating has been dropped along with php templates.
+
+Twig is now a required dependency and only templating engine this library supports.
```
Comment (ElectricMaxxx) [inline UPGRADE.md]: shouldn't we add some deprecation notices then on a minor release on 1.x? And we should add a line here how to fix consequences for a user.

Comment (kunicmarko20) [reply UPGRADE.md]: done in #133

**Comment by ElectricMaxxx** [comment]
please rebase to see changes on master.

**Comment by kunicmarko20** [comment]
@ElectricMaxxx any chance you are able to review/merge this and #117?

**Comment by ElectricMaxxx** [comment]
Thanks a lot

**Pull request merged**